src/main/java/com/minecolonies/coremod/colony/buildings/BuildingBuilder.java

```diff
@@ -126,7 +126,7 @@ public AbstractJob createJob(final CitizenData citizen)
     @Override
     public boolean neededForWorker(@Nullable final ItemStack stack)
     {
-        return Utils.isMiningTool(stack);
+        return Utils.isMiningTool(stack) || neededResources.containsKey(stack.getUnlocalizedName());
```

Comment:
True, i already told him to just overrule me on this one and I made a note on my famous list.